test/PoolFactory.js

```diff
@@ -2,42 +2,48 @@ const { expect } = require("chai");
 const { ethers } = require("hardhat")
 
 describe("PoolFactory contract", function () {
-  let poolFactoryContract;
-  let poolFactoryToken;
+  let alice;
+  let bob;
 
   beforeEach(async function () {
+    [alice, bob] = await ethers.getSigners();
+
+    libFactory = await ethers.getContractFactory("LibToken");
+    LibToken = await libFactory.deploy("LibToken", "Lib");
+    await LibToken.deployed();
+
+
+    skLibFactorty = await ethers.getContractFactory("StkLibToken")
+    skLibToken = await skLibFactorty.deploy(LibToken.address, "stkLib Token",  "stkLib")
+    await skLibToken.deployed();
+
     poolFactoryContract = await ethers.getContractFactory("PoolFactory");
-    [address1, admin1, admin2] = await ethers.getSigners();
+    poolFactoryInstance = await poolFactoryContract.deploy(skLibToken.address);
 
-    poolFactoryToken = await poolFactoryContract.deploy();
-    await poolFactoryToken.deployed();
+    await poolFactoryInstance.deployed();
   });
 
-  
-  describe("PoolFactory dynamics", function () {
```

Comment:
Why did you remove this? We need it to test the PoolFactory functions

Reply:
fixed

Reply:
Still looks deleted to me

Reply:
@kaleb I have integrated it back, I just added the test into a different testing that is alreay added

Reply:
Ohh, sounds good